components/SwipeRow.js

```diff
@@ -164,13 +164,16 @@ class SwipeRow extends Component {
 	}
 
 	manuallySwipeRow(toValue) {
-		Animated.spring(this.state.translateX,
+		Animated.spring(
+      this.state.translateX,
 			{
 				toValue,
 				friction: this.props.friction,
 				tension: this.props.tension
 			}
-		).start();
+		).start(function(){
+      this.props.onRowDidClose && this.props.onRowDidClose();
```

Comment:
It looks like this is being called whenever a row is manuallySwiped. So it's being triggered whether a row is openedOrClosed. I'd suggest checking the `toValue` and seeing if it's `0` before calling `onRowDidClose()`